Sources/Sentry/include/HybridPublic/SentrySessionReplayIntegration-Hybrid.h

```diff
@@ -16,4 +18,24 @@
 
 @end
 
+@interface
+SentrySessionReplayIntegration ()
+
++ (id<SentryRRWebEvent>)createBreadcrumbwithTimestamp:(NSDate *)timestamp
+                                             category:(NSString *)category
+                                              message:(NSString *)message
```

Comment:
message should be nullable here, same as in SentryRRWebBreadcrumbEvent initWithTimestmap